2 - RTB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -596,60 +596,59 @@ Come risultato dell'implementazione del processo di Gestione dei Rischi:
++ *Fornitura del profilo di rischio rilevante agli stakeholder in base alle loro esigenze*:\
+  il profilo di rischio viene all'occorrenza discusso nei meeting interni e/o esterni e tutte le parti interessate allo stato attuale dei rischi e delle azioni di trattamento possono consultare il documento Analisi dei Rischi v1.0.0. 
 
 
-==== Analizzare i rischi
+==== Analisi dei rischi
 Questa attività consiste nei seguenti compiti:
 
-+ identificare i rischi nelle categorie descritte nel contesto della Gestione dei Rischi:\ l'identificazione avviene durante tutte le fasi di sviluppo, inoltre, i rischi emergono dall'analisi delle misurazioni di qualità dei processi e del sistema software in evoluzione;
-
-+ stimare la probabilità di occorrenza e le conseguenze di ciascun rischio identificato:\ per ogni rischio identificato, viene stimata la probabilità di occorrenza e le conseguenze, che tipicamente coinvolgono impatti tecnici, di pianificazione, di costo o di qualità;
++ *identificazione del rischio ed associazione ad uno specifico profilo, secondo quanto descritto nella pianificazione della Gestione dei Rischi*:\ l'identificazione avviene durante tutte le fasi di sviluppo. Inoltre, i rischi emergono dall'analisi delle misurazioni di qualità dei processi e del sistema software in evoluzione;
 
-+ valutare ciascun rischio rispetto alle sue soglie di accettazione:\ ogni rischio viene valutato rispetto alle soglie stabilite, il superamento delle quali attiva delle modalità di trattamento specifiche.
++ *valutazione di ciascun rischio rispetto alle sue soglie di accettazione*:\ ogni rischio viene valutato rispetto alle soglie stabilite, il superamento delle quali attiva delle modalità di trattamento specifiche.
 
-==== Trattare i rischi
+==== Trattamento dei rischi
 Questa attività consiste nei seguenti compiti:
 
-+ per ciascun rischio che non soddisfi la sua soglia di tolleranza, definire e registrare le strategie e le misure di trattamento consigliate. Queste possono includere:\
++ *definizione e registrazione delle strategie e delle misure di trattamento consigliate per ciascun rischio che superi la soglia di tolleranza*. Queste possono includere:\
   - l'eliminazione del rischio;
   - la riduzione della sua probabilità o gravità;
   - l'accettazione del rischio.
 
   Vengono anche registrate informazioni sull'efficacia delle alternative di trattamento.
 
-+ quando viene accettato un rischio che non soddisfa la sua soglia, va considerato di priorità elevata e monitorato continuamente per determinare se sono necessarie future azioni di trattamento del rischio o se la sua priorità è cambiata;
++ *nel caso di accettazione di un rischio al di sopra della soglia, attribuzione di priorità elevata e monitoraggio costante per valutare la necessità di futuri interventi di trattamento del rischio o eventuali modifiche della sua priorità*;
```

Comment:
per rendere questo non troppo diverso dagli altri metterei in corsivo qualcosa come "attribuzione di priorità e monitoraggio:" e dopo i due punti, non in grassetto, metterei quello che adesso è in grassetto.

Reply:
"attribuzione di priorità e monitoraggio:" l'ho messa in grassetto per coerenza con gli elementi di sopra, dimmi se va bene. Il corsivo secondo le norme viene utilizzato solo per:
- citazioni;
- formati;
- nomi di practice;
- nomi propri di dominio.

Reply:
si scusami volevo dire grassetto invece di corsivo